src/app/catalogManagement/buyerCatalogs/templates/buyerCatalogs.html

```diff
@@ -66,6 +61,11 @@ <h3 class="l-page-header__title">
                             <i class="fa fa-caret-down" ng-show="buyerCatalogs.parameters.sortBy == 'Name'"></i>
                             <i class="fa fa-caret-up" ng-show="buyerCatalogs.parameters.sortBy == '!Name'"></i>
                         </th>
+                        <th rowspan="2" ng-click="buyerCatalogs.updateSort('CategoryCount')">
+                            Category Count
+                            <i class="fa fa-caret-down" ng-show="buyerCatalogs.parameters.sortBy == 'CategoryCount'"></i>
+                            <i class="fa fa-caret-up" ng-show="buyerCatalogs.parameters.sortBy == '!CategoryCount'"></i>
+                        </th>
                         <th rowspan="2">
```

Comment:
I get this message when I try to sort by category count:

{"Errors":[{"ErrorCode":"List.InvalidSortProperty","Message":"Property is not sortable.","Data":{"Property":"Catalog.CategoryCount"}}]}

Doesn't look like its possible